src/Microsoft.SqlTools.ServiceLayer/Scripting/ScriptAsOptions.cs

```diff
@@ -0,0 +1,712 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlTools.ServiceLayer.Scripting.Contracts;
+using Microsoft.SqlTools.Utility;
+
+namespace Microsoft.SqlTools.ServiceLayer.Scripting
+{
+    /// <summary>
+    /// The options for Script as operation
+    /// </summary>
```

Comment:
Can you add a comment that this class is mainly an adapter to map property names?